src/main/java/org/wildfly/common/cpu/ProcessorInfo.java

```diff
@@ -1,7 +1,6 @@
-package org.wildfly.common.cpu;
 /*
  * JBoss, Home of Professional Open Source.
- * Copyright 2015 Red Hat, Inc., and individual contributors
+ * Copyright 2017 Red Hat, Inc., and individual contributors
```

Comment:
Should we just make it, gasp, 2024?

Reply:
This is verbatim from the original file. I don't know if updating the copyright date has any implication; if so, we'd want to be correct, and if not, it doesn't matter anyway...